src/model/document/document.ts

```diff
@@ -242,7 +235,11 @@ function useElementFocus() {
 export function useDocument<TElements extends QuantumDocumentElementTypes<readonly QuantumElementType[]>>(
   elementTypes: TElements
 ): UseQuantumDocument<TElements> {
-  const gridCellSize = readonly(new Vector2(20, 20))
+  const options = reactive<DocumentOptions>({
+    gridCellSize: readonly(new Vector2(20, 20)),
+    paperStyle: 'standard',
+    paperSize: 'Legal',
```

Comment:
I almost want to make this `A4` by default, mostly because that's what I'm used to. It honestly doesn't matter very much either way. What do you think?

Reply:
I agree, Letter is a US sheet size. In fact, I don't think I have ever even used legal. In fact, US Letter size is equal to US Ansi A. What are your opinions on adding other sheet sizes, A5, A3, etc? https://papersizes.io/

Reply:
A5 and A3 are probably worth supporting as well (doesn't cost very much effort at all and would be nice test cases). I suppose the only reason to not support every single paper size in the world would be to spare the user the pain of having to search through a massive dropdown.

Reply:
Okay, I added a couple of other sizes and made A4 default.